src/module/actor/sheet/base.ts

```diff
@@ -418,12 +427,34 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
 
         // Sell treasure item
         $html.find(".item-sell-treasure").on("click", async (event) => {
```

Comment:
missed a spot for killing the jqueries

Reply:
That part should look like this then?
```ts
for (const sellButton of htmlQueryAll(html, ".item-sell-treasure")) {
sellButton.addEventListener("click", (event) => {
```

Reply:
More or less yeah. I'm feeling this pattern happens often enough to justify a new helper somehow but until then yeah.

Reply:
Done, thanks!